src/screencast/pipewire_screencast.c

```diff
@@ -0,0 +1,226 @@
+#include "pipewire_screencast.h"
+
+static inline void init_type(struct pwr_type *type, struct pw_type *map) {
```

Comment:
No need for `inline`, the compiler is better at deciding whether inlining is necessary